Expensify/react-native
# Inline code block support

# Upstream PR Link


## Summary


New `textCodeBlock` prop for a <Text /> component which is used for displaying inline code-blocks. Similar to a `<pre>` tag which supports customizable inline background color and border.

```js
const textCodeBlockStyle = {
  backgroundColor: '#002E22',
  borderColor: '#1B5744',
  borderRadius: 4,
  borderWidth: 2,
};

<Text textCodeBlockStyle={textCodeBlockStyle}>
  &nbsp;This text.&nbsp;
</Text>
```

See [Libraries/Text/TextCodeBlock.js](https://github.com/Expensify/react-native/pull/43/files#diff-6ad9e1a26acf75ca0a50607666f39f6064656f5e4cfec502e0376a833093e431) for the list of available props.

See [packages/rn-tester/js/examples/Text/TextExample.ios.js](https://github.com/Expensify/react-native/pull/43/files#diff-a251d41b798854ebdc060194e6ee3c2be1b8ac811bb6f8d1f2f2666136964334) for a usage example.

## Changelog


[iOS] [Added] - textCodeBlock prop for <Text /> component
[Android] [Added] - textCodeBlock prop for <Text /> component

## Test Plan



- Run rn-tester app on both iOS and Android 
- Navigate to Text Component on home screen
- Search for **Inline code-block** section
- Each block wrapped in a `<Text textCodeBlockStyle>` should have be displayed inline, with a background color and a border around it.
